### DISCUSSION TITLE
Financial-Grade API (FAPI 1.0) support

### DISCUSSION THREAD

**Discussion by eddyariki** [ory/hydra#3833] [Q&A] [answered]
FAPI is now a `Final Specification` in [Open ID Foundation](https://openid.net/wg/fapi/specifications/). 

Will Ory Hydra support FAPI anytime soon? 

Note: [this](https://community.ory.sh/t/hydra-fapi-support/1739) forum post already discusses this, but it's an old post, so I would like to confirm again.

**Comment by mig5** [comment]
Very interested in this too. FAPI 1 has been stable for a long time, FAPI 2 I understand to be the future of the standard and is currently in final stages of review by the OIDF.

My (early) understanding of FAPI is that it's basically a subset of OIDC/OAuth2.0 with some assumptions built in e,g mandating specific key lengths etc, use of state/nonce, probably a lot that Hydra already does out of the box. But also other stuff like restrictions on access token TTLs etc beyond the 'norm' or typical use case, and other details.

**Comment by aeneasr** [comment, marked as answer]
While FAPI is on our backlog it is highly unlikely to be addressed before we have commercial demand for it as it’s quite a lot of work! So generally a yes but an unknown to when. If your company requires it and you want it in Hydra please talk to us: www.ory.sh/contact
